tests/framework/ROM/TimeSeries/SyntheticHistory/tests

```diff
@@ -41,4 +41,19 @@
       zero_threshold = 1e-3
     [../]
   [../]
+
+  [./Wavelet]
+    type = 'RavenFramework'
+    input = 'wavelet.xml'
+    [./csv]
+      type = OrderedCSV
+      output = 'Wavelet/samples_0.csv Wavelet/samples_1.csv'
+      rel_err = 2e-1
+    [../]
+    [./xml]
+      type = XML
+      output = 'Wavelet/romMeta.xml'
+      rel_err = 1e-2
```

Comment:
we should be able to get the rel err down nearer to 1e-6 for this deterministic signal, right? Or is there a lot of numerical instability in the characterization?

Reply:
No, we can definitely put it lower. I'll do that.